### PR TITLE
Changes from $request->get() to correct methods part 7

### PR DESCRIPTION
**Pull request by adudekyeti**
Merge this after #9178 (getMultiDimensionArray change)

### CODE REVIEW

File: modules/CustomView/actions/Save.php [hunk 1]
```diff
@@ -64,15 +64,15 @@ private function getCVModelFromRequest(\App\Request $request)
...
 			'setmetrics' => $request->isEmpty('setmetrics') ? 0 : $request->getInteger('setmetrics'),
 			'status' => $request->getInteger('status', 0),
 			'featured' => $request->getInteger('featured', 0),
-			'color' => $request->get('color'),
-			'description' => $request->get('description'),
+			'color' => !$request->isEmpty('color') ? $request->getByType('color', 'Color') : '',
+			'description' => $request->getByType('description', 'Text'),
 		];
-		$selectedColumnsList = $request->get('columnslist');
+		$selectedColumnsList = $request->getArray('columnslist');
```
Comment (TomaszQr) [inline modules/CustomView/actions/Save.php]: $request->getArray('columnslist', 'Text')

File: modules/Settings/Menu/actions/SaveAjax.php [hunk 2]
```diff
@@ -20,7 +20,7 @@ public function __construct()
 
 	public function createMenu(\App\Request $request)
 	{
-		$data = $request->get('mdata');
+		$data = $request->getArray('mdata', 'Text');
```
Comment (TomaszQr) [inline modules/Settings/Menu/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/Menu/actions/SaveAjax.php [hunk 3]
```diff
@@ -34,7 +34,7 @@ public function createMenu(\App\Request $request)
 
 	public function updateMenu(\App\Request $request)
 	{
-		$data = $request->get('mdata');
+		$data = $request->getArray('mdata', 'Text');
```
Comment (TomaszQr) [inline modules/Settings/Menu/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/Menu/actions/SaveAjax.php [hunk 4]
```diff
@@ -62,7 +62,7 @@ public function removeMenu(\App\Request $request)
 	public function updateSequence(\App\Request $request)
 	{
 		$recordModel = Settings_Menu_Record_Model::getCleanInstance();
-		$recordModel->saveSequence($request->get('mdata'), true);
+		$recordModel->saveSequence($request->getArray('mdata', 'Text'), true);
```
Comment (TomaszQr) [inline modules/Settings/Menu/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/PickListDependency/actions/SaveAjax.php [hunk 5]
```diff
@@ -12,14 +12,14 @@ class Settings_PickListDependency_SaveAjax_Action extends Settings_Vtiger_Index_
...
-		$targetField = $request->get('targetField');
+		$sourceModule = $request->getByType('sourceModule', 'Alnum');
+		$sourceField = $request->getByType('sourceField', 'Alnum');
+		$targetField = $request->getByType('targetField', 'Alnum');
 		$recordModel = Settings_PickListDependency_Record_Model::getInstance($sourceModule, $sourceField, $targetField);
 
 		$response = new Vtiger_Response();
 		try {
-			$result = $recordModel->save($request->get('mapping'));
+			$result = $recordModel->save($request->getArray('mapping', 'Text'));
```
Comment (TomaszQr) [inline modules/Settings/PickListDependency/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/SalesProcesses/actions/SaveAjax.php [hunk 6]
```diff
@@ -16,7 +16,7 @@ public function __construct()
 
 	public function updateConfig(\App\Request $request)
 	{
-		$param = $request->get('param');
+		$param = $request->getArray('param', 'Standard');
```
Comment (TomaszQr) [inline modules/Settings/SalesProcesses/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/SupportProcesses/actions/SaveAjax.php [hunk 7]
```diff
@@ -16,7 +16,7 @@ public function __construct()
 
 	public function updateConfig(\App\Request $request)
 	{
-		$param = $request->get('param');
+		$param = $request->getArray('param', 'Text');
```
Comment (TomaszQr) [inline modules/Settings/SupportProcesses/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/TimeControlProcesses/actions/SaveAjax.php [hunk 8]
```diff
@@ -10,7 +10,7 @@ class Settings_TimeControlProcesses_SaveAjax_Action extends Settings_Vtiger_Basi
 {
 	public function process(\App\Request $request)
 	{
-		$params = $request->get('param');
+		$params = $request->getArray('param', 'Standard');
```
Comment (TomaszQr) [inline modules/Settings/TimeControlProcesses/actions/SaveAjax.php]: getMultiDimensionArray

File: modules/Settings/Menu/actions/SaveAjax.php (hunk 4 above)
Comment (adudekyeti) [reply modules/Settings/Menu/actions/SaveAjax.php]: Can not be "getMultiDimensionArray" because of nesting.

**Pull request merged**